assests/javascript/index.js

```diff
@@ -0,0 +1,73 @@
+const baseUrl = "http://127.0.0.1:3030";
+
+const newTask = document.getElementById("task-input");
+const addTaskIcon = document.getElementById("add-task-icon");
+const taskBox = document.getElementById("task-box");
```

Comment:
This should be `add-task-box` I think?

Reply:
yesssss. such a stupid mistake I did. 
Fixed!